2023-07-20 15:49:49,005 INFO    MainThread:13353 [internal.py:wandb_internal():90] W&B internal server running at pid: 13353, started at: 2023-07-20 15:49:49.005093
2023-07-20 15:49:49,006 DEBUG   HandlerThread:13353 [handler.py:handle_request():141] handle_request: run_start
2023-07-20 15:49:49,006 INFO    WriterThread:13353 [datastore.py:open_for_write():75] open: /home/nicole/marl-emecom/src/experiments_pgg_v0/anastassacos/wandb/offline-run-20230720_154948-2ajy617p/run-2ajy617p.wandb
2023-07-20 15:49:49,008 DEBUG   HandlerThread:13353 [meta.py:__init__():35] meta init
2023-07-20 15:49:49,008 DEBUG   HandlerThread:13353 [meta.py:__init__():49] meta init done
2023-07-20 15:49:49,009 DEBUG   HandlerThread:13353 [meta.py:probe():209] probe
2023-07-20 15:49:49,009 DEBUG   HandlerThread:13353 [meta.py:_setup_git():199] setup git
2023-07-20 15:49:49,017 DEBUG   HandlerThread:13353 [meta.py:_setup_git():206] setup git done
2023-07-20 15:49:49,017 DEBUG   HandlerThread:13353 [meta.py:_save_pip():53] save pip
2023-07-20 15:49:49,017 DEBUG   HandlerThread:13353 [meta.py:_save_pip():67] save pip done
2023-07-20 15:49:49,017 DEBUG   HandlerThread:13353 [meta.py:probe():247] probe done
2023-07-20 15:49:54,929 WARNING MainThread:13353 [internal.py:wandb_internal():153] Internal process interrupt: 1
2023-07-20 15:49:55,194 WARNING MainThread:13353 [internal.py:wandb_internal():153] Internal process interrupt: 2
2023-07-20 15:49:55,195 ERROR   MainThread:13353 [internal.py:wandb_internal():156] Internal process interrupted.
2023-07-20 15:49:55,931 INFO    WriterThread:13353 [datastore.py:close():279] close: /home/nicole/marl-emecom/src/experiments_pgg_v0/anastassacos/wandb/offline-run-20230720_154948-2ajy617p/run-2ajy617p.wandb
2023-07-20 15:49:55,931 INFO    HandlerThread:13353 [handler.py:finish():806] shutting down handler
2023-07-20 15:49:56,010 INFO    SenderThread:13353 [sender.py:finish():1111] shutting down sender
2023-07-20 15:49:56,011 INFO    MainThread:13353 [internal.py:handle_exit():80] Internal process exited
